WIP: Community Apps Page

swainn opened the pull request:
- Add a new page for community-contributed apps and add links to it in navigation.
- Update Project Road Map link to point at GitHub milestones.
- Address some divergent changes in the navigation and header across pages.
- Add the Tethys App Store as the first community-contributed app

swainn commented:
@sdc50 and @shawncrawley feedback on this would be appreciated. Content, layout, idea.

rileyhales commented:
Here's a DOI for the tethys app store paper in environmental modeling and software https://doi.org/10.1016/j.envsoft.2021.105227

rileyhales commented:
I think this looks good. The format/layout works fine until the number of apps listed grows substantially but i wouldn't worry about that problem until you have to. I would probably list the author as Rohit (maybe link to LinkedIn or something?) and then say something like supported/maintained by the BYU Hydroinformatics lab. The link to the journal article could be a button or a link like the name/license links, i don't think i have a preference on that.

swainn commented:
@rileyhales thanks for the feedback. Here's my responses. Screenshot above updated to show changes I made.

> I think this looks good. The format/layout works fine until the number of apps listed grows substantially but i wouldn't worry about that problem until you have to.

Good point on the format. I'm thinking only exceptional apps with broad application would be promoted on the Tethys website. The Tethys App Store should be the primary way of finding apps.

> I would probably list the author as Rohit (maybe link to LinkedIn or something?) and then say something like supported/maintained by the BYU Hydroinformatics lab.

I listed Rohit as the author and used his GitHub profile as the link. I changed BYU to be listed as the "sponsor".

> The link to the journal article could be a button or a link like the name/license links, i don't think i have a preference on that.

I added the publication as a button for (1) balance, seems to fill that space nicely, and (2) the title of most publications is too long to display by the author, sponsor, license--it throws of the balance.
